project-base/src/Shopsys/ShopBundle/Resources/graphql-types/Pagination/Connection.types.yml

```diff
@@ -0,0 +1,4 @@
+Connection:
+    type: object
```

Comment:
Why it is not relay-connection type? See this https://github.com/overblog/GraphQLBundle/blob/master/docs/definitions/relay/connection.md